### ISSUE TITLE
PM2.io servise is dead?

### ISSUE THREAD

**Issue by GROL** [Unitech/keymetrics.io#237]
I'm trying to pay and it was unsuccessful, support is not responding. 
Dead? Pity...

**Comment by cahva** [comment]
@GROL yeah been trying to contact them couple of months ago using the support plus via email but no answers. And now it seems their Zendesk support is not working anymore.
